src/procedure/nodes/context.h

```diff
@@ -39,14 +38,16 @@ class ProcedureContext
     void setConfiguration(Configuration *cfg);
     // Return target Configuration
     Configuration *configuration() const;
-    // Set target data list and prefix
-    void setDataListAndPrefix(GenericList &list, std::string_view prefix);
+    // Set prefix
+    void setPrefix(std::string_view prefix);
```

Comment:
With the removal of the `GenericList` argument the intention of this function becomes a little vague - I'd suggest we rename it:
```suggestion
    // Set prefix for generated processing data
    void setProcessingDataPrefix(std::string_view prefix);
```